.github/workflows/ci-cd.yml

```
@@ -0,0 +1,249 @@
+        
+      - name: Download build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: build-files
+          path: build/
+          
+      - name: Run E2E tests
+        run: npm run e2e:headless
+        env:
+          CYPRESS_baseUrl: http://localhost:3000
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)